### PR TITLE
POI - added single tap

### PR DESCRIPTION
**Pull request by crimean**

### CODE REVIEW

File: OsmAnd/src/net/osmand/plus/views/OsmandMapLayer.java [hunk 1]
```diff
@@ -30,6 +30,23 @@ public void onRetainNonConfigurationInstance(Map<String, Object> map) {}
 	
 	public void populateObjectContextMenu(Object o, ContextMenuAdapter adapter) {}
 
+	/**
+	 * return true when this layer should handle single tap instead of context layer
+	 * @return
+	 */
+	public boolean isSingleTapResponder() {
```
Comment (vshcherb) [inline OsmAnd/src/net/osmand/plus/views/OsmandMapLayer.java]: We can remove that method if we just follow onSingleTap single action click. onSingleTap returns true

File: OsmAnd/src/net/osmand/plus/views/OsmandMapLayer.java [hunk 2]
```diff
@@ -30,6 +30,23 @@ public void onRetainNonConfigurationInstance(Map<String, Object> map) {}
...
+		return false;
+	}
+
+	/**
+	 * return true when context layer should open context menu of the long pressed point
+	 * (instead of common behaviour when lat/lon text opens on long press)
+	 * @return
+	 */
+	public boolean shouldProcessLongPress() {
```
Comment (vshcherb) [inline OsmAnd/src/net/osmand/plus/views/OsmandMapLayer.java]: Similar suggestion to isSingleTapResponder(), so we don't overload map layer with methods.

Comment (crimean) [inline OsmAnd/src/net/osmand/plus/views/OsmandMapLayer.java]: If we do so - Measurement plugin will broke since it's layer lies under Context Menu layer.

**Comment by vshcherb** [comment]
can't we change order of layers?

**Pull request closed without merging**